config/complete/orderly-web.yml

```diff
@@ -69,15 +69,27 @@ volumes:
 
 # Optional: to migrate the underlying orderly metadata to outpack metadata
 outpack:
+  repo: mrcide
```

Comment:
this is a breaking change but moved the repo to the top level of `outpack`/`packit` to avoid uneccessary repetition. tbh I think all the containers in this whole config are (and will always be?) from the same repo so could maybe refactor this further.

Reply:
Yeah, agreed. This was before I'd realised we'd need a new tool!